validation/init.go

```diff
@@ -20,11 +20,18 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
+
 	StandardConfig["sepolia"] = new(StandardConfigTy)
 	err = decodeTOMLFileIntoConfig("standard-config-sepolia.toml", StandardConfig["sepolia"])
 	if err != nil {
 		panic(err)
 	}
+
+	StandardConfig["sepolia-dev-0"] = new(StandardConfigTy)
+	err = decodeTOMLFileIntoConfig("standard-config-sepolia-dev-0.toml", StandardConfig["sepolia-dev-0"])
```

Comment:
I'm including this at the moment for completeness, I do not believe any checks are actually using it presently.